frontend/src/pages/Traces/TracesList.tsx

```diff
@@ -311,7 +319,7 @@ const TracesTableRow = React.memo<TracesTableRowProps>(
 		return (
 			prevProps.virtualRowKey === nextProps.virtualRowKey &&
 			prevProps.isSelected === nextProps.isSelected &&
-			prevProps.gridColumns === nextProps.gridColumns
+			isEqual(prevProps.gridColumns, nextProps.gridColumns)
```

Comment:
Updated this to do deep equal check and prevent re-renders do to save by reference differences